src/components/ScrollBox/ScrollBox.module.css

```
@@ -0,0 +1,24 @@
+@supports selector(::-webkit-scrollbar) {
+  .scroller::-webkit-scrollbar {
+    width: 13px;
+    height: 13px;
+    background-color: transparent;
+    border-radius: 100px;
+  }
+
+  .scroller:hover::-webkit-scrollbar {
+    background-color: var(--mantine-color-dark-8);
+  }
+
+  .scroller::-webkit-scrollbar-thumb {
+    border-radius: 100px;
+    background-color: var(--mantine-color-dark-2);
```

```suggestion
    background-color: var(--mantine-color-dark-3);
```
? :) I still think it "pops" a bit much against the other elements